index.html

```diff
@@ -34,9 +34,9 @@ <h2 id="subtitle">WebSocket Sub-protocol</h2>
   <section id="abstract">
     <p>This document defines a WebSocket sub-protocol called the Web Thing
       Protocol, for monitoring and controlling connected devices over the
-      World Wide Web. The Web Thing Protocol is intended as a dedicated 
-      real-time protocol for the <a href="https://www.w3.org/WoT/">Web of 
-      Things</a>, to enable a WoT
+      World Wide Web. The Web Thing Protocol is intended as a dedicated
```

Comment:
One question.
Why is it called Web Thing Protocol and not Web of Things Protocol? or WoT Protocol?
What was the reason behind it?

Reply:
It is originally the protocol used by the WebThings gateway (https://webthings.io/)

Reply:
@egekorkan wrote:
> It is originally the protocol used by the WebThings gateway (https://webthings.io/)

That's not actually quite true. WebThings originally implemented Mozilla's [Web Thing API](https://webthings.io/api/), which does have a WebSocket API but that has different messages and only covers a tiny subset of the WoT operations which exist today. We haven't implemented the Web Thing Protocol Strawman Proposal anywhere in WebThings yet.

Essentially the [Web Thing REST API](https://webthings.io/api/#web-thing-rest-api) is being superceded by the [HTTP Basic Profile](https://w3c.github.io/wot-profile/#http-basic-profile),  and the [Web Thing WebSocket API](https://webthings.io/api/#web-thing-websocket-api) is being superceded by the Web Thing Protocol, but both are quite different to the original Web Thing API which predates all the current WoT specifications.

It's called the Web Thing Protocol because it's a protocol for communicating with [web things](https://www.w3.org/TR/wot-architecture/#sec-web-thing) :) (and that's slightly less long winded than "Web of Things Protocol").

Reply:
Thx